pkg/pkg.ml

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ocaml
+#use "topfind"
+#require "topkg"
+open Topkg
+
+
+let () =
+  Pkg.describe "tuntap" @@ fun c ->
+  Ok [
+    Pkg.mllib "lib/tuntap.mllib";
+    Pkg.clib "lib/libtuntap_stubs.clib";
+    Pkg.test "test/getifaddrs_test";
```

Comment:
I looked around (in nocrypto/ptime/mirage-solo5): you need to add dependencies that `test/*` depends on `tuntap_stubs.o`... this commit https://github.com/hannesm/ocaml-tuntap/commit/ecdfe02374b6898473f7af0eeadf725bc8f61f97 seems to work (although the test _before your topkg port_ do not run smooth on my system (compile and error out); neither do the tests after your topkg port ;)